docker-jenkins-sandbox

```diff
@@ -0,0 +1,13 @@
+FROM jenkins/jenkins:lts
+
+# deploy jenkins and jenkins plugins
+ENV JAVA_OPTS="-Djenkins.install.runSetupWizard=false"
+COPY jenkins_sandbox/plugins.txt /usr/share/jenkins/ref/plugins.txt
+RUN /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
+
+# install jenkins-job-builder
+USER root
+RUN apt-get update
+RUN apt-get install -y python python-dev python-pip
+RUN pip install --upgrade pip
+RUN pip install jenkins-job-builder
```

Comment:
Do you think it'd be worthwhile installing jenkins-job-builder using [constraints.txt](https://github.com/rcbops/rpc-gating/blob/master/constraints.txt) so the version used inside here is the same as we're using in "prod"?